rockcraft.yaml

```diff
@@ -24,24 +24,26 @@ parts:
     mysql-repo:
         plugin: nil
         override-pull: |
+            export DEBIAN_FRONTEND=noninteractive
+            export MYSQL_APT_CONFIG=0.8.23-1
             apt-get update
             apt-get install -y wget lsb-release gnupg
-            export MYSQL_APT_CONFIG=0.8.23-1
             wget -q https://dev.mysql.com/get/mysql-apt-config_${MYSQL_APT_CONFIG}_all.deb \
                 -O /tmp/mysql-apt-config_${MYSQL_APT_CONFIG}_all.deb
             dpkg -i /tmp/mysql-apt-config_${MYSQL_APT_CONFIG}_all.deb
             apt-get update
-    mysql-deb:
+    mysql:
         plugin: nil
         after: [mysql-repo]
-        override-stage: |
-            apt install -y \
-              mysql-shell \
-              mysql-server-8.0 \
-              util-linux
+        stage-packages:
+            - mysql-shell
+            - mysql-server-8.0
+            - util-linux
+        stage-snaps:
+            - xtrabackup/latest/edge
```

Comment:
can we add an issue to this repository to switch xtrabackup snap to the latest/stable channel once it has been released on stable?